composer.json

```diff
@@ -15,7 +15,6 @@
     "require-dev": {
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
         "phpcompatibility/php-compatibility": "^9.3",
-        "phpunit/phpunit": "^8.5",
```

Comment:
This was accidentally committed in a previous patch, and causes failures on the 5.6 - 7.1 builds when present (as that version requires 7.2+).